src/main/java/be/cytomine/client/models/Description.java

```diff
@@ -42,6 +42,14 @@ public Description(String domain, Long idDomain, String description) {
         set("domainIdent",idDomain.toString());
         set("data", description);
     }
+    public Description addDescription(Long domainIdent, String domainClassName, String text) throws CytomineException
```

Comment:
I don't get the need of this method. It's the role of the constructor to create a new object, see the code just above.
The old `Cytomine.addDescription(...)` method is replaced by `new Description(...).save()`

Reply:
Done